static/news/CHANGES.md

```diff
@@ -5,30 +5,32 @@
 **New Features**:
 
 - Added option to change web root in config/general
-- Added HorribleSubs, AniDex and Shanaproject torrent provider
+- Added HorribleSubs, AniDex and Shanaproject torrent providers
 - Added sorting and filtering in 'Missed Subtitles in PostProcess' page for users with 'Postpone PP if no subtitle' enabled
+- Added a centralized improved date parser for parsing of the publishing date for the providers: Limetorrents, Torrentz2, HDSpace, TorrentProject, GFTracker, HoundDawgs, Bit-HDTV, CPasBien
```

Comment:
Added a centralized improved publishing date parser for Limetorrents, Torrentz2, HDSpace, TorrentProject, GFTracker, HoundDawgs, Bit-HDTV, CPasBien